stdlib/private/OSLog/IntegerFormatting.swift

```diff
@@ -0,0 +1,317 @@
+@frozen
+public struct IntegerFormatting: Equatable {
```

Comment:
I'd pick a specific name to avoid potential future conflicts or shadowing like OSLogIntegerFormatting